packages/rum-core/src/domain/resource/matchRequestTiming.ts

```diff
@@ -2,13 +2,17 @@ import type { Duration, RelativeTime } from '@datadog/browser-core'
 import { addDuration } from '@datadog/browser-core'
 import type { RumPerformanceResourceTiming } from '../../browser/performanceCollection'
 import type { RequestCompleteEvent } from '../requestCollection'
-import { toValidEntry } from './resourceUtils'
+import { isValidEntry } from './resourceUtils'
 
 interface Timing {
   startTime: RelativeTime
   duration: Duration
 }
 
+// we use a WeakMap because WeakSet is not supported in ie11
+const PLACEHOLDER = 1
+const matchedResourceTimingEntries = new WeakMap<PerformanceEntry, typeof PLACEHOLDER>()
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about renaming it to `alreadyMatchedEntries`?
- `resourceTiming` seems superfluous as it is not used in `sameNameEntries`
- adding the `already` prefix could make the intent clearer